src/main/events/torrenting/resume-game-download.ts

```diff
@@ -1,6 +1,6 @@
 import { registerEvent } from "../register-event";
 import { GameStatus } from "../../constants";
-import { gameRepository } from "../../repository";
+import {gameRepository, userPreferencesRepository} from "../../repository";
```

Comment:
Are you using prettier? Please use prettier as a formatter or run `npm run format` before uploading
There should be whitespaces in between the brackets

Reply:
Done ✅